From 11e92de1808451e2811a38ca8e398457574b2f84 Mon Sep 17 00:00:00 2001
From: Fedora Bluez maintainers <bluez-owner@fedoraproject.org>
Date: Sun, 29 Mar 2015 18:28:48 +0200
Subject: [PATCH] fix invalid conversion

error: invalid conversion from 'const void*' to 'const uint8_t* {aka
const unsigned char*}' [-fpermissive]
---
 lib/bluetooth.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/bluetooth.h b/lib/bluetooth.h
index 6ca64b6..852a6b2 100644
--- a/lib/bluetooth.h
+++ b/lib/bluetooth.h
@@ -345,8 +345,8 @@ typedef struct {
 
 static inline void bswap_128(const void *src, void *dst)
 {
-	const uint8_t *s = src;
-	uint8_t *d = dst;
+	const uint8_t *s = (const uint8_t *) src;
+	uint8_t *d = (uint8_t *) dst;
 	int i;
 
 	for (i = 0; i < 16; i++)
-- 
2.3.4

